br/pkg/lightning/backend/kv/session.go

```diff
@@ -264,6 +264,15 @@ func newSession(options *SessionOptions, logger log.Logger) *session {
 	vars.SQLMode = sqlMode
 	if options.SysVars != nil {
 		for k, v := range options.SysVars {
+			// since 6.3(current master) tidb checks whether we can set a system variable
+			// lc_time_names is a read-only variable for now, but might be implemented later.
+			if sv := variable.GetSysVar(k); sv == nil {
+				logger.DPanic("unknown system var", zap.String("key", k))
+				continue
+			} else if sv.ReadOnly {
+				logger.Info("skip read-only variable", zap.String("key", k))
+				continue
+			}
```

Comment:
We'd better not set `lc_time_names`. Otherwise, there are still many unnecessary logs.

Reply:
as explained in comments, not sure whether tidb will support it in later release

Reply:
Can we reduce duplicate logs? In my test, importing 7G data will print 304 line logs like this.

Reply:
changed to debug, see https://github.com/pingcap/tidb/pull/37301/commits/632455456bc19de09949b846300ef01bc7446f07